CMakeLists.txt

```diff
@@ -84,3 +84,9 @@ if(NOVELRT_BUILD_TESTS)
 
   add_subdirectory(tests)
 endif()
+
+install(
+  EXPORT NovelRT
+  NAMESPACE NovelRT::
```

Comment:
Do we need the double colon here?

Reply:
I believe not necessarily but it may help? I framed this off of Fabulist's export

Reply:
I see, ok, well what do I know 😆

Reply:
The namespace is what it generates as the prefix in the CMake find_package scripts, for grouping all of the targets from a specific library together. For example:
```cmake
# Create imported target fabulist::lua
add_library(fabulist::lua STATIC IMPORTED)

set_target_properties(fabulist::lua PROPERTIES
  INTERFACE_COMPILE_DEFINITIONS "LUA_ANSI"
  INTERFACE_COMPILE_OPTIONS "\$<\$<CXX_COMPILER_ID:MSVC>:/experimental:external>;\$<\$<CXX_COMPILER_ID:MSVC>:/external:I>;\$<\$<CXX_COMPILER_ID:MSVC>:/external:W0>;\$<\$<CXX_COMPILER_ID:MSVC>:/external:templates->"
  INTERFACE_INCLUDE_DIRECTORIES "\$<\$<NOT:\$<CXX_COMPILER_ID:MSVC>>:>"
  INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "\$<\$<NOT:\$<CXX_COMPILER_ID:MSVC>>:>"
)
```